CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 * Update KafkaConnector CR status so the 'NotReady' condition is added if the connector or any tasks are reporting a 'FAILED' state.
 * Add auto-approval mechanism on KafkaRebalance resource when an optimization proposal is ready
+* Update Strimzi Kafka Bridge to 0.22.1
```

Comment:
Maybe you can remove it here as this should go into 0.31.1

Reply:
Ah ok. I wasn't aware of the plan to cherry pick. Instead of deleting should I add a 0.31.1 section in the CHANGELOG and having the line there?

Reply:
Maybe not, I guess you have other PRs on the pipeline for 0.31.1 with a section in the CHANGELOG. We should anyway remember to add this line later.

Reply:
It is a CVE. So we should release it with the patch release I think. I would not add a 0.31.1 section at this point. We can add it all together once we know the whole payload.